ddl/multi_schema_change.go

```diff
@@ -56,18 +57,135 @@ func (d *ddl) MultiSchemaChange(ctx sessionctx.Context, ti ast.Ident) error {
 	return d.callHookOnChanged(job, err)
 }
 
-func checkMultiSchemaInfo(info *model.MultiSchemaInfo, t table.Table) error {
-	err := checkOperateSameColAndIdx(info)
-	if err != nil {
-		return err
+func onMultiSchemaChange(w *worker, d *ddlCtx, t *meta.Meta, job *model.Job) (ver int64, err error) {
+	if job.MultiSchemaInfo.Revertible {
+		// Handle the rolling back job.
+		if job.IsRollingback() {
```

Comment:
Is our rollback test the next PR?

Reply:
Also, should we add some precise tests that test the behavior of each `subJob` and each of its states?

Reply:
The `TestCancel` is updated for the rollback test. Moreover, the tests about cancelling adding columns will be introduced in the next PR.

> Also, should we add some precise tests that test the behavior of each subJob and each of its states?

Yes. This will be added to the test-infra.